src/index.html

```diff
@@ -17,11 +21,360 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
     />
   </head>
+  <body class="page">
+    <header class="header">
+      <a href="#HOME">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link nav__link--is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the all links
```suggestion
          </li>
          
          <li class="nav__item">
```